### PR TITLE
Chore: Give the help information of Makefile to make it more clear

### PR DESCRIPTION
**Pull request by cubxxw**
Signed-off-by: Xinwei Xiong <3293172751nss@gmail.com>### Description of your changes

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested
```bash 
Usage:

  make [comment]

Comments:

  test                           Run tests
  test-cli-gen                   Run the unit tests for cli gen
  unit-test-core                 Run the unit tests for core
  build                          Build vela cli binary
  build-cli                      Clean build
  vet                            Run go vet against code
  staticcheck                    Run the staticcheck
  lint                           Run the golangci-lint
  build-swagger                  Build the swagger
  image-cleanup                  Delete Docker images
  image-load                     load docker image to the k3d cluster
  image-load-runtime-cluster     Load the run-time cluster image
  core-test                      Run tests
  manager                        Build vela core manager and apiserver binary
  vela-runtime-rollout-manager   Build vela runtime rollout manager binary
  manifests                      Generate manifests e.g. CRD, RBAC etc.
  check-license-header           Check license header
  def-gen                        Install definitions
  helm-doc-gen                   Generate helm chart README.md
  help                           Display help information
```

### CODE REVIEW

File: pkg/addon/addon_suite_test.go [hunk 1]
```diff
@@ -191,9 +191,9 @@ var _ = Describe("Addon test", func() {
 			return nil
 		}, 30*time.Second, 300*time.Millisecond).Should(BeNil())
 
-		notExsitAppName, err := determineAddonAppName(ctx, k8sClient, "not-exist")
+		notExitAppName, err := determineAddonAppName(ctx, k8sClient, "not-exist")
 		Expect(err).Should(BeNil())
-		Expect(notExsitAppName).Should(BeEquivalentTo("addon-not-exist"))
```
Comment (chivalryq) [inline pkg/addon/addon_suite_test.go:196]: There seems no reason for this change.

**Comment by chivalryq** [comment]
Please rebase the code. LGTM after rolling back the comment above.

**Comment by cubxxw** [comment]
> Please rebase the code. LGTM after rolling back the comment above.

Thanks for the review. I agree with you

**Review by wonderflow — COMMENTED** [review]
great job! please don't break the go mod.

File: Makefile [hunk 2]
```diff
@@ -124,12 +133,28 @@ HOSTARCH := amd64
...
+
+
+## help: Display help information
+help: Makefile
+	@echo ""
+	@echo "Usage:"
+	@echo ""
+	@echo "  make [comment]"
+	@echo ""
+	@echo "Comments:"
```
Comment (charlie0129) [inline Makefile:157]:
```suggestion
	@echo "  make [targets]"
	@echo ""
	@echo "Targets:"
```

Comment (cubxxw) [reply Makefile]: I agree ~

**Comment by Somefive** [comment]
Great contribution! Please rebase and fix the conflict. :)

**Comment by cubxxw** [comment]
> Great contribution! Please rebase and fix the conflict. :)

Thank you for your review.

**Comment by Somefive** [comment]
@cubxxw Sorry for the late reply. Seems the CI breaks and the Makefile has some conflicts. We have just reduced some CI processes to make it easier to pass. Could you try to rebase it again? P.S. We strongly recommend using rebase instead of merge for catching up main branches.

File: Makefile [hunk 3]
```diff
@@ -36,78 +44,79 @@ fmt: goimports installcue
 	$(CUE) fmt ./pkg/stdlib/pkgs/*
 	$(CUE) fmt ./pkg/stdlib/op.cue
 	$(CUE) fmt ./pkg/workflow/tasks/template/static/*
-# Run go vet against code
 
+## sdk_fmt: Run go fmt against code
```
Comment (chivalryq) [inline Makefile:48]:
```suggestion
## sdk_fmt: Check if SDK template reviewable
```

File: Makefile [hunk 4]
```diff
@@ -36,78 +44,79 @@ fmt: goimports installcue
...
 staticcheck: staticchecktool
 	@$(INFO) staticcheck
 	@$(STATICCHECK) $(shell go list ./...|grep -v scaffold)
 
+## lint: Run the golangci-lint
 lint: golangci
 	@$(INFO) lint
 	@$(GOLANGCILINT) run --skip-dirs 'scaffold'
 
+## reviewable: Run the reviewable
```
Comment (chivalryq) [inline Makefile:67]:
```suggestion
## reviewable: Check if the code reviewable. Make sure to run this before submit the PR.
```

File: Makefile [hunk 5]
```diff
@@ -36,78 +44,79 @@ fmt: goimports installcue
...
 	/bin/sh hack/e2e/build_runtime_rollout.sh
 	docker build -t $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE) -f runtime/rollout/e2e/Dockerfile.e2e runtime/rollout/e2e/
 	rm -rf runtime/rollout/e2e/tmp
 	k3d image import $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)  || { echo >&2 "kind not installed or error loading image: $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)"; exit 1; }
 	k3d cluster get $(RUNTIME_CLUSTER_NAME) && k3d image import $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE) --cluster=$(RUNTIME_CLUSTER_NAME) || echo "no worker cluster"
 
-# Run tests
+## core-test: Run tests
 core-test:
 	go test ./pkg/... -coverprofile cover.out
```
Comment (chivalryq) [inline Makefile:109]: Let's remove this unused target. It's duplicated with unit-test-core.

File: Makefile [hunk 6]
```diff
@@ -125,11 +134,26 @@ HOSTARCH := amd64
 endif
 
 
+## check-license-header: Check license header
 check-license-header:
 	./hack/licence/header-check.sh
 
+## def-gen: Install definitions
```
Comment (chivalryq) [inline Makefile:141]:
```suggestion
## def-gen: Install definitions to current Kubernetes cluster context.
```

**Comment by chivalryq** [comment]
@cubxxw Hi, besides all the suggested changes above. You could use `/retest foo` in a comment to restart the test that fails. Example: https://github.com/kubevela/kubevela/pull/5950#issuecomment-1530741460

**Comment by chivalryq** [comment]
@cubxxw Note the spell. It should be `e2e-multicluster-test`. And also I guess you modified the first comment. You can check why it's not restart here: https://github.com/kubevela/kubevela/actions/workflows/issue-commands.yml

File: Makefile [hunk 7]
```diff
@@ -36,78 +44,79 @@ fmt: goimports installcue
...
-
+	ifneq (, $(shell which docker))
+		ifneq ($(shell docker images -q $(VELA_CORE_TEST_IMAGE)),)
+			@docker rmi -f $(VELA_CORE_TEST_IMAGE)
+		endif
 
+		ifneq ($(shell docker images -q $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)),)
+			@docker rmi -f $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)
+		endif
+	endif
```
Comment (chivalryq) [inline Makefile:92]: You break the makefile grammar here. `ifneq` and such primitives should have no spaces ahead. This breaks the tests.

File: Makefile (hunk 3 above)
Comment (chivalryq) [reply Makefile:48]: @cubxxw ping, here's some suggested changes

**Comment by cubxxw** [comment]
@chivalryq thanks for ur help

**Review by Somefive — APPROVED** [review]
LGTM

**Review by chivalryq — APPROVED** [review]

**Review by wangyikewxgm — APPROVED** [review]

**Pull request merged**